src/Card/CardImageCap.jsx

```diff
@@ -47,7 +47,10 @@ const CardImageCap = React.forwardRef(({
 
   const handleSrcFallback = (event, altSrc) => {
     const { currentTarget } = event;
-    if (currentTarget.src !== altSrc) {
+
+    if (!altSrc || currentTarget.src.endsWith(altSrc)) {
+      currentTarget.style.display = 'none';
+    } else if (currentTarget.src !== altSrc) {
```

Comment:
It looks like this approach ends up rendering an empty logo div in the `Card` briefly before this logic hides it, leading to a brief flicker of this empty, white logo div.

I wonder if we could flip this logic around a bit? Keep the logo image hidden until we know we loaded an image properly, and then toggle the logo div to be visible; otherwise, keep it hidden.

Curious to hear your two cents @PKulkoRaccoonGang :)

Reply:
Example: 

[pgn-card-fallback-img-review.webm](https://user-images.githubusercontent.com/2828721/216644321-2d6e2ff3-6e11-4976-889e-fc906701d44b.webm)

Reply:
Yes, this is misbehavior. Thanks 👍